tests/Unit/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.py

```diff
@@ -12,13 +12,13 @@
     spacetime_normal_vector)
 
 
-def weight_function(coords, r_max):
+def spatial_weight_function(coords, r_max):
     r2 = np.sum([coords[i]**2 for i in range(len(coords))])
     return np.exp(-r2 / r_max / r_max)
 
 
-def spacetime_deriv_weight_function(coords, r_max):
-    W = weight_function(coords, r_max)
+def spacetime_deriv_spatial_weight_function(coords, r_max):
```

Comment:
Why not keep signatures consistent between this implementation of `spacetime_deriv_spatial_weight_function`, the return-by-ref implementaion in `DampedHarmonic.cpp`, and the return-by-value wrapper in tests below? It seems unnecessary and a bit confusing to me, but perhaps I am not seeing the intent here..

Reply:
My intent was to minimize the amount of code changes in the code the C++ is being compared to so as to make it easier to remain confident that the code still works after the changes. I have no general preference either way :)